semantic/src/site/globals/site.overrides

```diff
@@ -2287,12 +2287,13 @@ a:hover {
   }
 }
 
-.template-buider-category-input {
+.template-builder-category-input {
   .flex-column-center-start();
-  margin: 3px;
-  padding: 20px;
+  margin-top: 20px;
+  padding: 15px;
   border-radius: @borderRadius;
-  background: @surfacesLight;
+  border: 1px solid @surfacesMed;
+
```

Comment:
Typos, and changed background of Category editor to white, otherwise the checkboxes don't blend into BG very nicely.

OLD:
![Screenshot 2023-06-15 at 11 28 48 AM](https://github.com/openmsupply/conforma-web-app/assets/5456533/59f19ee7-1d17-4be7-989b-7db8ab6f0c69)

NEW:
![Screenshot 2023-06-15 at 11 27 36 AM](https://github.com/openmsupply/conforma-web-app/assets/5456533/6eb3bfe0-42a6-4194-a87b-e2b8b5e55e46)